dipy/tracking/utils.py

```diff
@@ -64,6 +64,8 @@
 from numpy import (asarray, ceil, dot, empty, eye, sqrt)
 from dipy.io.bvectxt import ornt_mapping
 from dipy.tracking import metrics
+from dipy.tracking.streamline import set_number_of_points
```

Comment:
I think this means that the CCI calculation is not a utility. I would put it in the streamlines module

Reply:
Thanks, @skoudoro and @arokem. Could one of you please explain this point a bit? I'm not following.

Reply:
I see. Thank you for explaining, @arokem. So this is a decision made about the structure of the tracking module as a whole. Utils gets imported by other tracking sub-modules but should only import from other modules (unless it's from `_utils` since that is lower on the hierarchy). I think this explains some other things I've been puzzled about debugging imports... I might be an offender with respect to this point so let me know if I can help fix the module if it's decided to enforce the hierarchy  : )

Reply:
I took your suggestion, @arokem, and moved the function to the streamlines sub-module in tracking